packages/components/components/elvis-app-bridge/src/react/elvia-app-bridge.tsx

```diff
@@ -0,0 +1,62 @@
+import { newTab } from '@elvia/elvis-assets-icons';
+import { ContextMenu } from '@elvia/elvis-context-menu/react';
+import { IconWrapper, TertiaryButton } from '@elvia/elvis-toolbox';
+import { Tooltip } from '@elvia/elvis-tooltip/react';
+import React, { useState } from 'react';
+
+import { AppIcon } from './AppIcon';
+import { AppBridgeProps } from './elvia-app-bridge.types';
+import { getLinks } from './getLinks';
+import { getActiveApp } from './utils';
+
+export const AppBridge: React.FC<AppBridgeProps> = function ({ className, inlineStyle, targetId, ...rest }) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const links = getLinks(targetId);
+  const activeApp = getActiveApp();
+
+  return (
+    <ContextMenu
+      onOpen={() => setIsOpen(true)}
+      onClose={() => setIsOpen(false)}
+      trigger={
+        <Tooltip
+          trigger={
+            <TertiaryButton isActive={isOpen}>
+              Åpne i
+              <IconWrapper icon={newTab} size="xs" />
+            </TertiaryButton>
+          }
+          content={'Åpne målpunktet i en annen applikasjon'}
+        />
```

Comment:
Should we disable the tooltip when the context menu is open?

Reply:
It doesn't hurt to keep it enabled IMO, and if I disable it when the menu is open it does a weird flashing-thing when the context menu closes 🥲